src/main/java/carpet/script/Context.java

```diff
@@ -22,6 +23,7 @@
     public final Map<String, LazyValue> variables = new HashMap<>();
 
     public final ScriptHost host;
+    public PlayerEntity s;
```

Comment:
don't change the contexts. The carpet context will have command source for you and you dont' need anything player related to convert dates.

Reply:
ok, I never did that